pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -165,43 +165,11 @@ type BundleStatus struct {
 	DefaultCAPackageVersion *string `json:"defaultCAVersion,omitempty"`
 }
 
-// BundleCondition contains condition information for a Bundle.
-type BundleCondition struct {
-	// Type of the condition, known values are (`Synced`).
-	Type BundleConditionType `json:"type"`
-
-	// Status of the condition, one of ('True', 'False', 'Unknown').
-	Status metav1.ConditionStatus `json:"status"`
-
-	// LastTransitionTime is the timestamp corresponding to the last status
-	// change of this condition.
-	// +optional
-	LastTransitionTime *metav1.Time `json:"lastTransitionTime,omitempty"`
-
-	// Reason is a brief machine readable explanation for the condition's last
-	// transition.
-	// +optional
-	Reason string `json:"reason,omitempty"`
-
-	// Message is a human readable description of the details of the last
-	// transition, complementing reason.
-	// +optional
-	Message string `json:"message,omitempty"`
-
-	// If set, this represents the .metadata.generation that the condition was
-	// set based upon.
-	// For instance, if .metadata.generation is currently 12, but the
-	// .status.condition[x].observedGeneration is 9, the condition is out of date
-	// with respect to the current state of the Bundle.
-	// +optional
-	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
-}
-
 // BundleConditionType represents a Bundle condition value.
 type BundleConditionType string
```

Comment:
suggestion: Is this type now redundant? If so I'd be comfortable removing it

Reply:
Yes, I just forgot to remove it. Thanks! Fix incoming.